### PR TITLE
Update aws, awsx, eks, and friends

### PR DESCRIPTION
**Pull request by thomas11**
Update all examples but five to latest aws, awsx, eks, and aws-apigateway. **This update is to the latest alpha versions directly - only for testing, not for merging**.

Also updated a few other dependencies along the way when I came across them.

I haven't fully figured out how to handle the breaking changes in these upgrades:
  - aws-ts-apigateway-auth0
  - aws-ts-lambda-efs
  - aws-ts-netlify-cms-and-oauth/cms-oauth/infrastructure
  - aws-ts-pulumi-webhooks
  - aws-ts-slackbot

### CODE REVIEW

**Comment by mnlumi** [comment]
@thomas11 The PR looks good, but I can't see why the one check is failing on destroy. Is this unrelated?

File: aws-py-hub-and-spoke-network/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 pulumi>=3.0.0,<4.0.0
-pulumi_aws>=5.0.0,<6.0.0
-pulumi_awsx>=1.0.0,<2.0.0
+pulumi_aws>=6.0.0,<7.0.0
+pulumi_awsx==2.0.0b3
```
Comment (scottslowe) [inline aws-py-hub-and-spoke-network/requirements.txt:3]: Is there a reason we're specifying what appears to be a beta release here? And pinning the version?

**Review by scottslowe — CHANGES_REQUESTED** [review]

File: aws-py-hub-and-spoke-network/requirements.txt (hunk 1 above)
Comment (thomas11) [reply aws-py-hub-and-spoke-network/requirements.txt:3]: An oversight, thanks!

**Review by scottslowe — APPROVED** [review]
Approved, thanks!

**Pull request merged**